packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -311,8 +311,8 @@ LiveLabel.defaultProps = {
   dir: 'ltr',
 };
 
-const StoryPromo = ({ image, info, mediaIndicator, topStory }) => (
-  <StoryPromoWrapper topStory={topStory}>
+const StoryPromo = ({ image, info, mediaIndicator, topStory, sectionName }) => (
+  <StoryPromoWrapper topStory={topStory} data-story-promo={sectionName}>
```

Comment:
for story promo, we should pass the section name and the url of the story

Reply:
I thought this would be something to be done in simorgh, where you would combine the story title and the url to form a `sectionName`.

Reply:
hmm yeah good point. Will make sure this is mentioned on the issue for implementing event tracking for this component